### PR TITLE
fix(orthanc): Correct bulkdata URL handling and add configuration example PDF

### PR DESCRIPTION
**Pull request by wayfarer3130**
### Context

Orthanc responds with an invalid URL for bulkdata, and wasn't sure about the configuration for retrieving bulkdata for PDF display.  This PR adds the ability to fix a broken URL when it is prefixed by a fixed string, and adds an example orthanc.js configuration file.

Fixing this https://github.com/OHIF/Viewers/issues/4256

### Changes & Results

Changed fixBulkDataURI to replace a startsWith setting with another fixed string.
Added orthanc.js as an example file

### Testing
Run a local orthanc server, and modify the webpack-pwa.js file to proxy data to it and then store an SR.  Ensure you can see the SR.

Bonus testing: Store a video file to orthanc and demonstrate that you can show the video.

#### PR

- [] My Pull Request title is descriptive, accurate and follows the
  semantic-release format and guidelines.

#### Code

- [] My code has been well-documented (function documentation, inline comments,
  etc.)

#### Public Documentation Updates

- [] The documentation page has been updated as necessary for any public API
  additions or removals.

#### Tested Environment

- [] OS: 
- [] Node version: 
- [] Browser:
  

[blog]: https://circleci.com/blog/triggering-trusted-ci-jobs-on-untrusted-forks/
[script]: https://github.com/jklukas/git-push-fork-to-upstream-branch

### CODE REVIEW

**Comment by sedghi** [comment]
@amazy

**Comment by amazy** [comment]
@sedghi @wayfarer3130 I confirm this solves our issue in our [test setup](https://github.com/orthanc-server/orthanc-setup-samples/commit/5af834d3d6e755e55d8d937547ace6fa39925626).

Huge thanks for your collaboration, this will make a lot of Orthanc users happy !

File: platform/app/public/config/orthanc.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        thumbnailRendering: 'wadors',
+        enableStudyLazyLoad: true,
+        supportsFuzzyMatching: false,
+        supportsWildcard: true,
+        staticWado: true,
+        singlepart: 'bulkdata',
+        bulkDataURI: {
+          enabled: true,
+          relativeResolution: 'studies',
+          // This is an example config that could potentially be used to fix the retrieve URL
```
Comment (sedghi) [inline platform/app/public/config/orthanc.js:30]: let's move this to local_orthanc

Comment (wayfarer3130) [reply platform/app/public/config/orthanc.js]: Done - also explained why the attributes are needed there.

**Review by sedghi — APPROVED** [review]

**Pull request merged**